Makefile

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Is the intention to build RedisAI starting from this Makefile in the future, even in non-RLEC builds?

Reply:
This Makefile is an intermediate step to the Makefile that will be introduced by the ARM branch.
The idea is twofold:
(1) Provide a convenient interface of invoking build/testing functionality,
(2) Move build logic away from .circleci/config.yml and Dockerfile into Makefiles and scripts.

So basically, packing logic introduced here in Makefile would move into its own script.
As for starting a build, I guess that one will simply invoke "make" which would then invoke cmake.

Reply:
I'm good with this strategy, especially since the arm64 build revamp is on its way.